gammapy/data/maps.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+
+from ..extern.bunch import Bunch
+from astropy.io import fits
+
+__all__ = ['MapsBunch']
+
+log = logging.getLogger(__name__)
+
+
+class MapsBunch(Bunch):
+    """
+    Minimal version of a future map container class.
```

Comment:
Instead of this sentence, put a sentence describing the purpose of this class is now.

Maybe add a code example loading like this to make it easy for people to play around with this?
```python
from gammapy.data import MapsBunch
maps = MapsBunch.read('$GAMMAPY_EXTRA/....something.fits')
```
Or, if you have that example in the high-level docs, maybe link to the high-level docs page from here?